src/backend/web/routes/posts.js

```diff
@@ -84,7 +85,22 @@ posts.get('/:id', async (req, res) => {
         message: `Post not found for id ${id}`,
       });
     } else {
-      res.json(post);
+      const accept = accepts(req);
+
+      switch (accept.type(['json', 'text', 'html'])) {
```

Comment:
I'm attempting to test this now (via [Postman](https://www.getpostman.com/)). From my understanding, if this code ([which I am currently checking out locally](https://imgur.com/Pvygcp8)) is working correctly, I should be able to send a GET request to `http://localhost:3000/posts` using an `Accept: text` header and receive a repsonse with `Content-Type: text/plan`.

I have attempted several such requests, but have only been receiving `application/json` content in response, e.g.

![image](https://user-images.githubusercontent.com/13500769/73199905-c1c76900-4103-11ea-8385-124125b02fe3.png)

I've tried multiple `Accept` directives (e.g. `text`, `text/plain`, `html`), but always receive the same JSON-formatted response.

@yatsenko-julia Were you able to receive a different result during your testing?
(@humphd I would greatly appreciate some guidance if I'm obviously going about testing this the wrong way, here!)

Reply:
@Silvyre I think this is for `posts/:id`, not `/posts`. Try postman again but requesting a specific post, and then you should get the text back. I'll do some testing myself too.

Reply:
@manekenpix Ah, `/posts/:id` appears to function as I expected:
![image](https://user-images.githubusercontent.com/13500769/73205995-0f49d300-4110-11ea-9c2d-11b288af8221.png)

(`Accept: text/plain` also worked fine for GET http://localhost:3000/posts/e69f9ca5df)

* * *

I may have misinterpreted what @humphd was requesting in https://github.com/Seneca-CDOT/telescope/issues/443#issuecomment-574774894; my previous understanding was that we wished for the `/posts` route to support the `Accept` header. Now that we appears to have the `Accept` header working `/posts/:id`, would it be useful if we implement the same functionality for `/posts`?

Reply:
@Silvyre `/posts` only returns an array of ids that can be used with `posts/:id`, I don't know if it'd be useful for that case.
For `/posts/:id`, this is what I got:
**HTML**
![html](https://user-images.githubusercontent.com/23108901/73207790-51c0df00-4113-11ea-85b1-e47652aec443.png)

**Text**
![text](https://user-images.githubusercontent.com/23108901/73207792-54233900-4113-11ea-86d4-5793764c354d.png)
:+1:

Reply:
Alright, cool.